layouts/brand.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div
+    class="group/layout"
+    :data-with-container="true"
+  >
+    <SiteHeader />
+    <BannerReadOnly />
+    <slot />
+    <div class="brand-layout">
+        <div class="brand-background"></div>
+    </div>
+    <SiteFooter />
+  </div>
+</template>
+
+<style scoped>
+.brand-layout {
+    top: 0;
+    width: 100vw;
```

Comment:
```suggestion
    width: 100%;
```

This is the line responsible for my lateral scroll issue. It's working for me with both width 100% or even no css for the width.